src/sdk/account/utils/Utils.ts

```diff
@@ -391,3 +396,32 @@ export type EthersWallet = {
   address: Address | string
   provider: AnyData
 }
+
+export const getAllowance = async (
+  client: PublicClient,
+  accountAddress: Address,
+  tokenAddress: Address
+): Promise<bigint> => {
+  const approval = await client.readContract({
+    address: tokenAddress,
+    abi: erc20Abi,
+    functionName: "allowance",
+    args: [accountAddress, BICONOMY_TOKEN_PAYMASTER]
+  })
+
+  return approval as bigint
```

Comment:
why casting here?

Reply:
I was seeing "unknown" type coming from readContract, now it seems like it does not complain anymore. Resolved.